joomla/joomla-cms
# [4.0] Cassiopeia - update card mod chrome

Pull Request for Issue #30678 (second attempt) .

### Summary of Changes
Updated html > layouts > chromes > card.php  (renamed from 'default', see #30729) to display mod-id as the containing element's (address, div, section, etc) `#id` instead of just in the title so that the id displays whether the title is shown or not.

### Testing Instructions

1. Create a new module if none already exists and in the Advanced tab select Cassiopeia's 'card' chrome from the list in 'Module Style' if not already applied.
2. On the front end, use the browser's inspector to inspect this module with title shown and then again where title is not shown
Result: See section "Actual result BEFORE applying this Pull Request" below.
3. Apply the patch of this PR
4. Repeat steps 1. and 2.
See section "Actual result AFTER applying this Pull Request" below.
### Actual result BEFORE applying this Pull Request
module `#id` only appears if title is shown

### Expected result AFTER applying this Pull Request
1. module `#id` always applied to Cassiopeia modules using Cassiopeia 'card' chrome
2. id can now be used for anchors, etc if 1. above applies.

No visible difference on front end for website visitors

### Documentation Changes Required
Note to highlight change
